changelogs/fragments/fix_nxos_zone_zoneset.yaml

```diff
@@ -0,0 +1,5 @@
+---
+bugfixes:
+  - "`nxos_zone_zoneset` - zone member addition with smart zoning in an already existing zone should be a no-op (https://github.com/ansible-collections/cisco.nxos/issues/339)."
+
```

Comment:
@srbharadwaj The extra empty lines are failing the sanity tests. Could you please remove them?
```
Running sanity test 'yamllint' with Python 3.6
2021-08-20 06:43:46.613766 | centos-8-stream | Run command: /usr/bin/python3.6 /root/ansible/test/lib/ansible_test/_data/sanity/yamllint/yamllinter.py
2021-08-20 06:44:05.383839 | centos-8-stream | ERROR: Found 1 yamllint issue(s) which need to be resolved:
2021-08-20 06:44:05.384186 | centos-8-stream | ERROR: changelogs/fragments/fix_nxos_zone_zoneset.yaml:5:1: empty-lines: too many blank lines (2 > 0)
2021-08-20 06:44:05.384254 | centos-8-stream | See documentation for help: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/yamllint.html
2021-08-20 06:44:05.385930 | centos-8-stream | ERROR: The 1 sanity test(s) listed below (out of 31) failed. See error output above for details.
2021-08-20 06:44:05.385984 | centos-8-stream | yamllint
```
You can run `ansible-test sanity --python <x.y>` locally before pushing the commit to verify.